app/Services/CO2viaCnaught.php

```diff
@@ -11,12 +12,12 @@ public function __construct(
     ) {
     }
 
-    public function getCO2InGrams(float $distanceInKm, float $massKg = 0): float
+    public function getCO2Footprint(float $distanceInKm, float $massKg = 0): CO2Footprint
     {
         $response = Http::withToken($this->apiKey)
             ->post('https://api.cnaught.com/v1/quotes/ride', ['distance_km' => $distanceInKm])
             ->json();
 
-        return $response['amount_kg'] * 1000;
+        return new CO2Footprint($response['amount_kg'] / 1000);
```

Comment:
Shouldn't this be a multiplier instead of division?

Reply:
You're right. I will delete the entire file.